spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -197,6 +199,53 @@
 
               expect(SocureDocvResultsJob).not_to have_received(:perform_later)
             end
+            context 'when document capture session exists' do
+              let(:docv_transaction_token) { 'fake-transaction-token' }
+              let(:user) { create(:user) }
+              let(:document_capture_session) do
+                DocumentCaptureSession.create(user:).tap do |dcs|
+                  dcs.socure_docv_transaction_token = docv_transaction_token
+                end
+              end
+
+              before do
+                request.headers['Authorization'] = socure_secret_key
+                allow(DocumentCaptureSession).to receive(:find_by).
+                  and_return(document_capture_session)
+                allow(SocureDocvResultsJob).to receive(:perform_later)
+                allow(RateLimiter).to receive(:new).with(
+                  {
+                    user: user,
+                    rate_limit_type: :idv_doc_auth,
+                  },
+                ).and_return(rate_limiter)
+                document_capture_session.socure_docv_capture_app_url = fake_capture_app_url
+                document_capture_session.save
+              end
+
+              it 'does not increment rate limiter of user' do
+                expect(rate_limiter.attempts).to eq 0
+                post :create, params: webhook_body
+                expect(rate_limiter.attempts).to eq 0
+                post :create, params: webhook_body
+                expect(rate_limiter.attempts).to eq 0
+              end
+
+              it 'resets socure_docv_capture_app_url to nil' do
+                expect(document_capture_session.socure_docv_capture_app_url).
+                  to eq(fake_capture_app_url)
+                post :create, params: webhook_body
+                document_capture_session.reload
+                expect(document_capture_session.socure_docv_capture_app_url).to be_nil
+              end
```

Comment:
all of the other test scenarios in this context already exist ... can you move this test to the existing context and rid of the rest of the context?

there is now a factory for DocumentCaptureSession .... pls follow the existing pattern

also:
- can you create a context for SESSION_EXPIRED event containing this same test
- please add a test to DOCUMENT_UPLOAD showing that the socure_docv_capture_app_url is not deleted from the document capture session